src/cpu/arm.rs

```diff
@@ -19,7 +19,8 @@
 
 #[cfg(all(
```

Comment:
Please add a comment above this along the following lines (with any typos fixed):
> uclibc: When linked statically, uclibc doesn't provide `getauxval`. When linked dynamically, recent versions do provide it, but we want to support older versions too. Assume that if uclibc is being used, this is an embedded target where the user cares a lot about minimizing code size and also that they know in advance exactly what target features are supported, so rely only on static feature detection.

Reply:
Done!